app/assets/javascripts/geoblacklight/modules/item.js

```diff
@@ -9,7 +9,5 @@ Blacklight.onLoad(function() {
     viewer = new window['GeoBlacklight']['Viewer'][viewerName](element);
   });
 
-  $('.truncate-abstract').readmore({
```

Comment:
Is this technically a breaking change? Is it possible to check which version is around and do the right thing based off that? (with a note to remove for GeoBlacklight 3.0?)

Reply:
Maybe its not a breaking change.. the interface I guess stays the same

Reply:
Yeah, no breaking change here. Just leaving the jQuery plugin version for a vanilla JS implementation. View markup doesn't change at all.